ports/zeldadoi/port.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": 3,
+    "name": "zeldadoi.zip",
+    "items": [
+      "Zelda Dungeons of Infinity.sh",
+      "zeldadoi"
+    ],
+    "items_opt": [],
+    "attr": {
+      "title": "The Legend of Zelda: Dungeons of Infinity",
+      "porter": [
+        "t0b10",

```

Comment:
I have a question about the porters.

Did all these rework the whole port or did write full patches or something or did they just help out with single things. Because we do have a credit and thank you section in the readme. Being a porter lays on support requests, requests to update the port and being responsible for the port. 

Example: 

I compile a port, and you make the whole control scheme via gptk. I'll list you as my coporter. 
I compile a port and you give me hints how to do something, you give me code snippet or you help me test it extensive, you get a special thank you note in the readme. 

If you did 95% of the work it does not warrant having 3 other Porters.

Reply:
We all OK'd it.